Bottom: 974a0ca9e74e54ef49653c6fcf9f18cb875591f7
Top:    e8a3c9257b2d13849705bef1c53f07561b43a360
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 14:49:21 +0100

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 2637495..94c6779 100644
--- a/configure.ac
+++ b/configure.ac
@@ -393,8 +393,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
